pandas/tests/io/parser/test_common.py

```diff
@@ -2124,5 +2124,13 @@ def test_no_header_two_extra_columns(all_parsers):
     ref = DataFrame([["foo", "bar", "baz"]], columns=column_names)
     stream = StringIO("foo,bar,baz,bam,blah")
     parser = all_parsers
-    df = parser.read_csv(stream, header=None, names=column_names, index_col=False)
-    tm.assert_frame_equal(df, ref)
+    with tm.assert_produces_warning(ParserWarning):
+        df = parser.read_csv(stream, header=None, names=column_names, index_col=False)
+        tm.assert_frame_equal(df, ref)
+
+
+def test_first_row_length(all_parsers):
+    stream = StringIO("col1,col2,col3\n0,1,2,X\n4,5,6\n6,7,8")
+    parser = all_parsers
+    with tm.assert_produces_warning(ParserWarning):
+        parser.read_csv(stream, index_col=False)
```

Comment:
Could you add the example from original issue (with the stray comma after the "6")